src/sentry/testutils/fixtures.py

```diff
@@ -374,7 +374,7 @@ def create_codeowners(self, project=None, code_mapping=None, **kwargs):
 
     def create_slack_integration(
         self,
-        organization: "Organization",
+        organization: Organization,
```

Comment:
If you look deeper into the calls you will see that is not a string, thus, the default was bad.